client/data/settings/selectors.js

```diff
@@ -39,6 +39,64 @@ export const getAccountStatementDescriptor = ( state ) => {
 	return getSettings( state ).account_statement_descriptor || '';
 };
 
+export const getAccountBusinessName = ( state ) => {
+	return getSettings( state ).account_business_name || '';
+};
+
+export const getAccountBusinessURL = ( state ) => {
+	return getSettings( state ).account_business_url || '';
+};
+
+export const getAccountBusinessSupportAddress = ( state ) => {
+	return getSettings( state ).account_business_support_address || '';
+};
+
+export const getAccountBusinessSupportAddressCountry = ( state ) => {
+	return getSettings( state ).account_business_support_address.country || '';
+};
+
+export const getAccountBusinessSupportAddressLine1 = ( state ) => {
+	return getSettings( state ).account_business_support_address.line1 || '';
+};
+
+export const getAccountBusinessSupportAddressLine2 = ( state ) => {
+	return getSettings( state ).account_business_support_address.line2 || '';
+};
+
+export const getAccountBusinessSupportAddressCity = ( state ) => {
+	return getSettings( state ).account_business_support_address.city || '';
+};
+
+export const getAccountBusinessSupportAddressPostalCode = ( state ) => {
+	return (
+		getSettings( state ).account_business_support_address.postal_code || ''
+	);
+};
+
+export const getAccountBusinessSupportEmail = ( state ) => {
+	return getSettings( state ).account_business_support_email || '';
+};
+
+export const getAccountBusinessSupportPhone = ( state ) => {
+	return getSettings( state ).account_business_support_phone || '';
+};
+
+export const getAccountBrandingLogo = ( state ) => {
+	return getSettings( state ).account_branding_logo || '';
+};
+
+export const getAccountBrandingIcon = ( state ) => {
+	return getSettings( state ).account_branding_icon || '';
+};
+
+export const getAccountBrandingPrimaryColor = ( state ) => {
```

Comment:
Perhaps we can set the default values for primary and secondary colors to some recommended colors? @LevinMedia may be able to offer some clues here.

Reply:
@anu-rock Definitely default colors need to be set, but in my opinion, they need to be set as a fallback on the backend.

Reply:
Also a follow up item.